src/content/reference/cache.md

```diff
@@ -0,0 +1,174 @@
+    -  A [`Response`](/reference/response) object to store under the given key.
+
+</Definitions>
+
+#### Invalid parameters
+
+`cache.put` throws an error if:
+  - the `request` passed is a method other than `GET`
+  - the `response` passed is a `status` of [`206 Partial Content`](https://httpstatuses.com/206)
+  - the `response` passed contains the header `Vary: *` (required by the Cache API specification)
+
+### `Match`
+
+```javascript
+cache.match(request, options)
+```
+
+<Definitions>
+
+- <Code>match(request, options)</Code> <TypeLink href="/reference/response">Promise{`<Response>`}</TypeLink>
+
+    - Returns a promise wrapping the response object keyed to that request. 
+
+</Definitions>
+
+#### Parameters
+
+<Definitions>
+
+- `request` <Type>string</Type>|<TypeLink href="/reference/request">Request</TypeLink>
+
+    - The string or [`Request`](/reference/apis/request) object used as the lookup key. Strings are interpreted as the URL for a new `Request` object.
+
+- `options`
+    -  Can contain one possible property: `ignoreMethod` (Boolean) Consider the request method a GET regardless of its actual value.
+
+</Definitions>
+
+Unlike the browser Cache API, Cloudflare Workers do not support the `ignoreSearch` or `ignoreVary` options on `match()`. You can accomplish this behavior by removing query strings or HTTP headers at `put()` time.
+
+Our implementation of the Cache API respects the following HTTP headers on the request passed to `match()`:
+
+<Definitions>
+
+- `Range`
+    - Results in a `206` response if a matching response is found. Your Cloudflare cache always respects range requests, even if an `Accept-Ranges` header is on the response.
+
+- `If-Modified-Since`
+    - Results in a `304` response if a matching response is found with a `Last-Modified` header with a value after the time specified in `If-Modified-Since`.
+
+- `If-None-Match`
+    - Results in a `304` response if a matching response is found with an `ETag` header with a value that matches a value in `If-None-Match`.
+
+- `cache.match()`
+    - Never sends a subrequest to the origin. If no matching response is found in cache, the promise that `cache.match()` returns is fulfilled with `undefined`.
+
+</Definitions>
+
+### `Delete`
+
+```javascript
+cache.delete(request, options)
+```
+
+<Definitions>
+
+- <Code>delete(request, options)</Code> <TypeLink href="/reference/response">Promise{`<boolean>`}</TypeLink>
+
+</Definitions>
+
+Deletes the `Response` object from the cache and returns a `Promise` for a Boolean response:
+
+- `true`: The response was cached but is now deleted
+- `false`: The response was not in the cache at the time of deletion.
+
+#### Parameters
+
+<Definitions>
+
+- `request` <Type>string</Type>|<TypeLink href="/reference/request">Request</TypeLink>
+
+    - The string or [`Request`](/reference/apis/request) object used as the lookup key. Strings are interpreted as the URL for a new `Request` object.
+
+<!-- What type is this? -->
```

Comment:
For comments like these, prefix with ”TODO” so they can be easily found and dealt with later.

```diff
- <!-- What type is this? -->
+ <!-- TODO: add `<Type/>` for this -->
```